outputs.tf

```diff
@@ -3,5 +3,10 @@
 # ------------------------------------------------------------------------------
 output "arn" {
   description = "The ARN of the certificate that is being validated."
-  value       = "${aws_acm_certificate_validation.main.certificate_arn}"
+  value       = "${aws_acm_certificate_validation.main.*.certificate_arn}"
+}
+
+output "wildcard_arn" {
+  description = "The ARN of the wildcard certificate that is being validated."
+  value       = "${aws_acm_certificate_validation.wildcard.*.certificate_arn}"
```

Comment:
What happens here if `var.wait_for_validation` is set to false?

Reply:
If wait for validation is set to false then the validation request is made as normal but the terraform script does not wait for confirmation that the validation is complete.  This means that the script cannot return the arn of the created certificate as an output.  It also means that you know the script will not hang and lock everything up.

Reply:
Sorry for not explaining myself better, if `var.wait_for_validation` is set to `false` this `${aws_acm_certificate_validation.wildcard.*.certificate_arn}`won't exist right? Will the terraform script break?

Reply:
Using the splat syntax prevents an error - if you set wait_for_validation to false you get:
cert_arn = []

Reply:
Good stuff, I was unsure how this works 👍